generate.sh

```diff
@@ -133,23 +195,24 @@ function CreateFramework {
         return
     fi
 
-    # x86_64
+    if [[ $is_maccatalyst == true ]]; then
+        fpath="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/iOSSupport/System/Library/Frameworks"
```

Comment:
You're assuming Xcode is in /Applications/Xcode.app here, I think something like this would be better:

```suggestion
        fpath="$(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/iOSSupport/System/Library/Frameworks"
```